internal/innosat/telecommand.go

```diff
@@ -0,0 +1,23 @@
+package innosat
+
+import (
+	"encoding/binary"
+	"io"
+)
+
+//TCDataFieldHeader (9 octets)
+type TCDataFieldHeader struct {
+	PUS            uint8
+	ServiceType    uint8
+	ServiceSubType uint8
+}
+
+// Read TCDataFieldHeader
+func (tcdfh *TCDataFieldHeader) Read(buf io.Reader) error {
+	return binary.Read(buf, binary.BigEndian, &tcdfh)
```

Comment:
how about a test of a real TC Data Field Header buffer from a `.rac` file resulting in the expected struct?